frontend/kesaseteli/employer/package.json

```diff
@@ -8,8 +8,9 @@
     "start": "NODE_ENV=production node start.js",
     "lint": "eslint --ext js,ts,tsx src",
     "pre-commit": "lint-staged -c ../../.lintstagedrc.js",
-    "test": "jest",
+    "test": "jest --runInBand --no-cache",
```

Comment:
run tests sequentially for kesaseteli. This is temporary solution. The problem is probably nock-library which does not work well when running tests in parallel. It need to be changed to something else